XRTK-Core/Packages/com.xrtk.core/Interfaces/SpatialAwareness/IMixedRealitySpatialAwarenessSystem.cs

```diff
@@ -81,6 +90,7 @@ public interface IMixedRealitySpatialAwarenessSystem : IMixedRealityEventSystem
         /// Sets the provided mesh <see cref="SpatialMeshDisplayOptions"/> on all the <see cref="DetectedSpatialObservers"/>
         /// </summary>
         /// <param name="displayOptions"></param>
+        [Obsolete("Use SpatialMeshVisibility instead")]
```

Comment:
Not sure whether using Obsolete / having obsolete APIs in a v0.1.5 Software that isn't bascially even released makes sense? Why not just get rid of it right away. What do you think?

Reply:
I'm fine with this I guess since I made it pretty clear in the Pull Request overview.